### PR TITLE
quick start: add an "explore more" section

### PR DESCRIPTION
**Pull request by yikeke**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -292,3 +292,16 @@ TiDB-Wasm 是运行在浏览器中的 TiDB 数据库，打开网页即可使用
         ```shell
         tiup cluster display <cluster-name>
         ```
+
+## 探索更多
+
+- 如果你刚刚部署好一套 TiDB 本地测试集群：
+    - 学习 [TiDB SQL 操作](/basic-sql-operations.md)
+    - [迁移数据到 TiDB](/data-migration-route.md)
+    - 了解 [TiDB 的核心特性与核心应用场景](/overview.md)
```
Comment (kissmydb) [inline quick-start-with-tidb.md:301]: 增加一个
了解[TiDB 整体架构](/tidb-architecture.md)

Comment (yikeke) [inline quick-start-with-tidb.md:301]:
```suggestion
    - 了解 [TiDB 的核心特性与核心应用场景](/overview.md)
    - 了解 [TiDB 的整体架构](/tidb-architecture.md)
```

Comment (yikeke) [reply quick-start-with-tidb.md:301]: Added in https://github.com/pingcap/docs-cn/pull/3486/commits/da331c34b14f08c1f535a883a22b8f0c09863c14

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -292,3 +292,17 @@ TiDB-Wasm 是运行在浏览器中的 TiDB 数据库，打开网页即可使用
         ```shell
         tiup cluster display <cluster-name>
         ```
+
+## 探索更多
```
Comment (lilin90) [inline quick-start-with-tidb.md:296]: 或者更直白一点儿呢？😀
```suggestion
## 接下来
```

Comment (yikeke) [reply quick-start-with-tidb.md:296]: 感觉有点奇怪，「接下来」是不是有点不太像标题？

Comment (lilin90) [reply quick-start-with-tidb.md:296]: 在中文里是会显得口语化一些，前后逻辑关系感觉会更强一些。可以先用当前版本。

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -294,3 +294,17 @@ TiDB-Wasm 是运行在浏览器中的 TiDB 数据库，打开网页即可使用
...
+    - 学习 [TiDB SQL 操作](/basic-sql-operations.md)
+    - [迁移数据到 TiDB](/data-migration-route.md)
+    - 了解 [TiDB 的核心特性与核心应用场景](/overview.md)
+    - 了解 [TiDB 的整体架构](/tidb-architecture.md)
+    - 了解 [TiDB 与 MySQL 的兼容性](/mysql-compatibility.md)
+
+- 如果你准备好在生产环境部署 TiDB 了：
+    - 在线部署：[使用 TiUP 部署 TiDB 集群](/production-deployment-using-tiup.md)
+    - 离线部署：[使用 TiUP 离线部署 TiDB 集群](/production-offline-deployment-using-tiup.md)
+    - [使用 TiDB Operator 在云上部署 TiDB](https://pingcap.com/docs-cn/tidb-in-kubernetes/stable/)
```
Comment (yikeke) [inline quick-start-with-tidb.md:310]: All the links should be aligned in https://github.com/pingcap/docs/pull/2728/files after all of the docs are ready. This PR needs a follow-up, so I tagged this PR as "translation/doing" instead. @ran-huang

Comment (ran-huang) [reply quick-start-with-tidb.md:310]: Links updated in https://github.com/pingcap/docs/pull/3300 😉

Comment (yikeke) [reply quick-start-with-tidb.md:310]: Good!

**Pull request merged**